src/types/mpi.rs

```diff
@@ -28,23 +29,36 @@ const MAX_EXTERN_MPI_BITS: u32 = 16384;
 /// );
 /// ```
 ///
-pub fn mpi(input: &[u8]) -> nom::IResult<&[u8], MpiRef<'_>> {
+pub fn mpi(input: &[u8]) -> IResult<&[u8], MpiRef<'_>> {
     let (number, len) = be_u16(input)?;
 
     let bits = u32::from(len);
     let len_actual = (bits + 7) >> 3;
 
     if len_actual > MAX_EXTERN_MPI_BITS {
-        Err(Err::Error(error_position!(
-            input,
-            nom::ErrorKind::Custom(errors::MPI_TOO_LONG)
-        )))
+        /*
+            Err(Err::Error(
+                error_position!(
+                input,
+                nom::error::ErrorKind::Custom(errors::MPI_TOO_LONG)
+            )))
+            FIXME should this give more information in the error?
+            FIXME is this the right Error Variant?
```

Comment:
what about these?

Reply:
I think originally this would end up as `pgp::errors::Error::ParsingError(nom::error::ErrorKind::Custom(MPI_TOO_LONG))` with MPI_TOO_LONG being just the constant `1000u32`.

`nom::error::ErrorKind` no longer has a `Custom` Variant, 
should I add new variant to `pgp::errors::Error`, keep it as is without the FIXME comment or something else?

Reply:
Invalid input is good enough for now.

Reply:
Ok the I will remove the FIXME